### PR TITLE
Add class to password field

### PR DESCRIPTION
**Pull request by JevgenijVisockij**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Class param doesn't work for password type. So added the class to the input.
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Should probably be tested by developer. You can test by adding a password field with class param to any fields_options form( be it module or PS core). Without this fix that field won't have an additional class added by class param and with the fix it does.

### CODE REVIEW

**Review by kpodemski — APPROVED** [review]

**Review by mflasquin — DISMISSED** [review]

**Comment by zuk3975** [comment]
I've tested it on AdminCustomerThreads controller which has password field. Seems to be alright :heavy_check_mark: 
I added "class" option in field_options:
Screenshot from 2022-12-06 17-19-18

Result without PR :x: :
Screenshot from 2022-12-06 17-14-09

Result with PR changes :heavy_check_mark: :
Screenshot from 2022-12-06 17-20-48

So the only question I have before merge, is it ok to have that extra "form-control" class? It wasn't there before, but it doesn't seem to cause any trouble. Seems like @JevgenijVisockij added it just to comply with other input like "text" type.
@kpodemski @mflasquin wdyt?

**Comment by mflasquin** [comment]
> I've tested it on AdminCustomerThreads controller which has password field. Seems to be alright heavy_check_mark I added "class" option in field_options: Screenshot from 2022-12-06 17-19-18
> 
> Result without PR x : Screenshot from 2022-12-06 17-14-09
> 
> Result with PR changes heavy_check_mark : Screenshot from 2022-12-06 17-20-48
> 
> So the only question I have before merge, is it ok to have that extra "form-control" class? It wasn't there before, but it doesn't seem to cause any trouble. Seems like @JevgenijVisockij added it just to comply with other input like "text" type. @kpodemski @mflasquin wdyt?

@zuk3975 Is the class form-control is here if you don't set an extra class ?

**Comment by zuk3975** [comment]
@mflasquin 
> Is the class form-control is here if you don't set an extra class ?

yes, as you can see here https://github.com/PrestaShop/PrestaShop/pull/30503/files#diff-b90e12704e4bf3a49b19ea07803b186e33bb03e5621dc4055bf93b071a3bc982R197, @JevgenijVisockij added the class manually. To respect the scope and lower the impact chances we should probably get rid of :thinking:

**Comment by kpodemski** [comment]
@JevgenijVisockij let's remove it, just to make sure

File: admin-dev/themes/default/template/helpers/options/options.tpl [hunk 1]
```diff
@@ -194,7 +194,7 @@
 										</div>
 									{elseif $field['type'] == 'password'}
 										<div class="col-lg-9">{if isset($field['suffix'])}<div class="input-group{if isset($field.class)} {$field.class}{/if}">{/if}
-											<input type="{$field['type']}"{if isset($field['id'])} id="{$field['id']}"{/if} size="{if isset($field['size'])}{$field['size']|intval}{else}5{/if}" name="{$key}" value=""{if isset($field['autocomplete']) && !$field['autocomplete']} autocomplete="off"{/if} />
+											<input class="form-control {if isset($field['class'])}{$field['class']}{/if}" type="{$field['type']}"{if isset($field['id'])} id="{$field['id']}"{/if} size="{if isset($field['size'])}{$field['size']|intval}{else}5{/if}" name="{$key}" value=""{if isset($field['autocomplete']) && !$field['autocomplete']} autocomplete="off"{/if} />
```
Comment (kpodemski) [inline admin-dev/themes/default/template/helpers/options/options.tpl]:
```suggestion
											<input{if !empty($field['class'])} class="{$field['class']}"{/if} type="{$field['type']}"{if isset($field['id'])} id="{$field['id']}"{/if} size="{if isset($field['size'])}{$field['size']|intval}{else}5{/if}" name="{$key}" value=""{if isset($field['autocomplete']) && !$field['autocomplete']} autocomplete="off"{/if} />
```

**Review by kpodemski — CHANGES_REQUESTED** [review]

**Comment by zuk3975** [comment]
removed the extra class added "form-control" just to make sure and retested.
QA :heavy_check_mark:

**Comment by zuk3975** [comment]
removed the extra class added "form-control" just to make sure and retested.
QA :heavy_check_mark: 
Screenshot from 2022-12-14 23-51-38
Screenshot from 2022-12-14 23-51-13

**Review by zuk3975 — APPROVED** [review]

**Review by FabienPapet — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
Thanks @JevgenijVisockij and @zuk3975 for the tests 👍

**Pull request merged**